test/base/test_concurrency_py3k.py

```diff
@@ -101,3 +107,36 @@ def go():
             await greenlet_spawn(go)
 
         await to_await
+
+    @async_test
+    @testing.requires.python37
+    @testing.skip_if(lambda: not hasattr(greenlet, "gr_context"))
```

Comment:
shouldn't this work in both cases, since we are blocking the 0.4.17 release?

Reply:
Ah, this is because we removed the code to support greenlet < 0.4.17, and this test didn't cover the part when it's not working with <0.4.17, so I added this skip to avoid a false "PASS".

Reply:
well technically, looking at the test, it just verifies that the context vars are properly set when using `await_only` and `greenlet_spawn` so I think it's not a false PASS for the older greenlet. It just happened to handle it that way without any other code change.

I think we could make it run so we don't accidentally break it for some other reason

Reply:
That makes sense, PR is updated with the latest master now 🎉 thanks for reviewing!